documentation/docs/get-started/api-server-get-started/basic-operations.md

```diff
@@ -270,3 +270,51 @@ To get the rendered VC, you can try this endpoint [`/renderCredential`](http://l
 :::tip
 The document value is a string which is a encoded HTML string. You can use any online [Base64](https://www.base64decode.org/) tool out there to decode it. Good luck!
 :::
+
+## Revoking a VC
+
+To revoke a VC, use the following VCKit API endpoint: [/agent/setBitstringStatus](http://localhost:3332/agent/setBitstringStatus).
```

Comment:
Hi, will we use only the Bitstring Status List or will we also support the Revocation list? I think this line would be good if only the Bitstring Status List were available.

Reply:
The `RevocationStatusList` is not compatible with VCDM v2, so we should only support the `BitstringStatusList` for now.